test/01-constructor.test.js

```diff
@@ -0,0 +1,27 @@
+const MyToken = artifacts.require("MyToken");
+
+contract("Constructor", accounts => {
```

Comment:
The name of the contract should be placed here not which you test. Also make one of the 3 files for test

Reply:
you can also use the following view instead of accounts:

"Constructor", [owner, user1, user2] => {